### PR TITLE
10 setup circleci

### PR DESCRIPTION
**Pull request by adixks**
Pls review my code.

### CODE REVIEW

**Comment by adixks** [comment]
Pls review my code :)

**Comment by adixks** [comment]
Pls review my code :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      # Use mvn clean and package as the standard maven build phase
+      - run:
+          name: Build
+          command: mvn -B -DskipTests clean package
+      # Then run your tests!
+      - run:
+          name: Test
+          command: mvn test
+      # Deployment on Server
+      - run:
```
Comment (JaRutkowski) [inline .circleci/config.yml:33]: Please do not use redeploy option. Please refer to the [`jfs-operations-cloud` page](https://github.com/JaRutkowski/jfs-operations-cloud/wiki/Resources-%F0%9F%93%81#circle-ci).

**Review by JaRutkowski — CHANGES_REQUESTED** [review]

**Comment by JaRutkowski** [comment]
Please merge (once u'll get the 🟢 light) the https://github.com/adixks/googleApi/pull/2#pullrequestreview-1362762730 first and then please create the PR for this, only with the changes specific to this task.

**Pull request merged**